.changeset/sharp-trees-yawn.md

```diff
@@ -0,0 +1,18 @@
+---
+"@comet/brevo-api": major
+"@comet/brevo-admin": minor
+---
+
+Added functionality to import Brevo contacts from CSV files.
+
+**Important Notice:**
+
+-   A major bump in `@comet/brevo-api` is required because a new `.env` variable named `REDIRECT_URL_FOR_IMPORT` is needed.
```

Comment:
Why is this redirect URL necessary? Why can't this be specified as CLI arg during the import?

Reply:
Yes I agree with what @thomasdax98.

@thomashuettmaier  and I already talked about it as well, and we also thought that it probably makes the most sense to define it this way. Even though I don't like defining another variable for redirectUrl, all the other solutions we came up with were worse.